scripts/e2e-monolith.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash -x
 
 # Monolith version to test against
-export MONOLITH_BUILDVERSION=2022.10.1-begin-254-gaefef87dfc4e
+# TODO: auto-select a recent stable version OKTA-561403
+export MONOLITH_BUILDVERSION=2022.12.2-begin-259-gdeb54c572369
```

Comment:
I've been using static URL paths to `whatever.domain/healthcheck` to get trex/preview/prod versions, not sure what you would want to default to though